UpgradeSpringBoot_2_0 recipe still using deprecated org.openrewrite.java.dependencies.UpgradeParentVersion

<!--
Thank you for reporting an issue with OpenRewrite!
We appreciate you taking the time to help us improve.
Please fill out the template below to help us understand and reproduce the issue.
Feel free to delete any sections that don't apply to your issue.
-->

## What version of OpenRewrite are you using?
<!--
Whenever possible please try to replicate your issue with the latest versions of OpenRewrite.
The latest major and minor versions of OpenRewrite projects are usually listed here:
https://docs.openrewrite.org/reference/latest-versions-of-every-openrewrite-module
For patch releases check the GitHub Releases page for the respective project.

We release every few weeks, so it's possible that your issue has already been fixed.

If you want to try the most recent changes that haven't been fully released yet, you can check out our snapshot releases
https://docs.openrewrite.org/reference/snapshot-instructions
-->
I am using rewrite-maven-plugin:5.0.0, rewrite-spring-5.0.0

## How are you running OpenRewrite?
<!--
Are you using the Maven plugin, Gradle plugin, Moderne CLI, Moderne.io or something else?
Is your project a single module or a multi-module project?

Can you share your configuration so that we can rule out any configuration issues?

Is your project public? If so, can you share a link to it?
Code snippets can also be shared privately via [our public Slack](https://join.slack.com/t/rewriteoss/shared_invite/zt-nj42n3ea-b~62rIHzb3Vo0E1APKCXEA).
-->
I am using the Maven plugin, by the command
```
 mvn -U org.openrewrite.maven:rewrite-maven-plugin:run "-Drewrite.recipeArtifactCoordinates=org.openrewrite.recipe:rewrite-spring:LATEST" "-Drewrite.activeRecipes=org.openrewrite.java.spring.boot2.UpgradeSpringBoot_2_7"
```

## What is the smallest, simplest way to reproduce the problem?
Any spring boot project

## What did you expect to see?
No specific error

## What did you see instead?
[ERROR] Recipe validation error in org.openrewrite.java.dependencies.UpgradeParentVersion: Recipe class org.openrewrite.java.dependencies.UpgradeParentVersion cannot be found
[ERROR] Recipe validation error in org.openrewrite.java.dependencies.UpgradeParentVersion: Recipe class org.openrewrite.java.dependencies.UpgradeParentVersion cannot be found
[ERROR] Recipe validation error in org.openrewrite.java.dependencies.UpgradeParentVersion: Recipe class org.openrewrite.java.dependencies.UpgradeParentVersion cannot be found

## What is the full stack trace of any errors you encountered?
N/A

## Are you interested in [contributing a fix to OpenRewrite](https://github.com/openrewrite/.github/blob/main/CONTRIBUTING.md#contributing-fixes)?
<!-- Indicate if this is something you would like to work on, and how we can best support you in doing so. -->
I think this should be easy to fix.

 [org.openrewrite.java.dependencies.UpgradeParentVersion](https://github.com/openrewrite/rewrite-spring/blob/main/src/main/resources/META-INF/rewrite/spring-boot-20.yml#L38) and [org.openrewrite.java.dependencies.UpgradeDependencyVersion](https://github.com/openrewrite/rewrite-spring/blob/main/src/main/resources/META-INF/rewrite/spring-boot-20.yml#L42) should be org.openrewrite.maven.UpgradeParentVersion and org.openrewrite.maven.UpgradeDependencyVersion instead.

--- a/src/main/resources/META-INF/rewrite/spring-boot-20.yml
+++ b/src/main/resources/META-INF/rewrite/spring-boot-20.yml
@@ -35,7 +35,7 @@ recipeList:
       artifactId: "*"
       newVersion: 2.0.x
       overrideManagedVersion: false
-  - org.openrewrite.java.dependencies.UpgradeParentVersion:
+  - org.openrewrite.maven.UpgradeParentVersion:
       groupId: org.springframework.boot
       artifactId: spring-boot-starter-parent
       newVersion: 2.0.x

